docs/en/pid-eaa-entity-configuration.rst

```diff
@@ -0,0 +1,141 @@
+.. include:: ../common/common_definitions.rst
+.. _Entity_Configuration_Credential_Issuer:
+
+Entity Configuration of PID/(Q)EAA Providers
+--------------------------------------------
+
+The PID/(Q)EAA Providers is a Federation Entity and, according to Section :ref:`Configuration of the Federation`, it MUST make a *well-known* endpoint available containing its Entity Configuration.
+The Entity Configuration of PID/(Q)EAA Providers MUST contain the claims as defined in Sections :ref:`Entity Configuration Leaves and Intermediates` and :ref:`Entity Configurations Common Parameters`. 
+
+The PID/(Q)EAA Providers MUST provide the following metadata types:
+
+  - `federation_entity`
+  - `oauth_authorization_server`
+  - `openid_credential_issuer`
+
+In case the PID/(Q)EAA Providers authenticate the Users using their Wallet Instance, then the metadata for *wallet_relying_party* MUST be provided in addition to the metadata above. In case a national eID scheme is used by the PID/(Q)EAA Providers for the User authentication, they MAY include a metadata for *openid_relying_party* within their Entity Configuration. The *openid_relying_party* metadata MUST be compliant with the current version of `SPID/CIE id OIDC Technical Specification <https://docs.italia.it/italia/spid/spid-cie-oidc-docs/it/versione-corrente/index.html>`_.
```

Comment:
```suggestion
In case the (Q)EAA Providers authenticate the Users using their Wallet Instance, then the metadata for *wallet_relying_party* MUST be provided in addition to the metadata above. In case a national eID scheme is used by the PID/(Q)EAA Providers for the User authentication, they MAY include a metadata for *openid_relying_party* within their Entity Configuration. The *openid_relying_party* metadata MUST be compliant with the current version of `SPID/CIE id OIDC Technical Specification <https://github.com/italia/spid-cie-oidc-docs>`_.
```